lucien-ios/AppDelegate.swift

```diff
@@ -18,7 +19,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         self.window = window
         window.rootViewController = RootViewController()
         window.makeKeyAndVisible()
+
+        // Initialize sign-in
+        GIDSignIn.sharedInstance().clientID = "1072472744835-miivpr72vpanvmpm2f3tbb7msae67tii.apps.googleusercontent.com"
+        GIDSignIn.sharedInstance().delegate = self
+
         return true
     }
 
+    private func application(application: UIApplication,
+                             openURL url: URL, options: [UIApplicationOpenURLOptionsKey: AnyObject]) -> Bool {
+        return GIDSignIn.sharedInstance().handle(url as URL!,
+                                                 sourceApplication: options[UIApplicationOpenURLOptionsKey.sourceApplication] as? String,
+                                                 annotation: options[UIApplicationOpenURLOptionsKey.annotation])
+    }
+
+    func sign(_ signIn: GIDSignIn!, didSignInFor user: GIDGoogleUser!, withError error: Error!) {
+        guard error == nil else { print("\(error.localizedDescription)"); return }
+        let userId = user.userID
+        let idToken = user.authentication.idToken
+        let fullName = user.profile.name
+        let givenName = user.profile.givenName
+        let familyName = user.profile.familyName
+        let email = user.profile.email
+        print(userId!, idToken!, fullName!, givenName!, familyName!, email!)
+    }
+
+    func sign(_ signIn: GIDSignIn!, didDisconnectWith user: GIDGoogleUser!, withError error: Error!) {
```

Comment:
The `GIDSignInDelegate` protocol marked this method as `@optional`; if no work's being done here, we can leave it out for now.